templates/NOTES.txt

```diff
@@ -24,3 +24,13 @@ You have installed {{ .Chart.Name }} as release {{ .Release.Name }} into namespa
     {{- println }}Routing to the kubernetes dashboard is provided at (https:/) /kubernetes/#/
 {{- end }}
 {{- end }}
+
+{{- if $.Values.global.redisClusterEnabled }}
+    {{- $clusterNodes := include "palisade.redis-cluster.headlessNodes" . | trimAll "," }}
+    {{- println }}Redis is clustered using nodes: {{ $clusterNodes }}
+{{- else }}
+    {{- $masterFullname := printf "%s-master" (include "palisade.redis.fullname" .) }}
+    {{- $masterPort := int .Values.global.redis.master.service.port }}
+    {{- println }}Redis is master/worker with master at: {{ $masterFullname }}:{{ $masterPort }}
+{{- end }}
```

Comment:
I'd like to see a warning here for the specific case when redis(cluster) is set to install but does not match the flag:
```
if (redisClusterEnabled and redis.install) or (not redisClusterEnabled and redisCluster.install)
println Redis install type does not match redis deployment type, are you sure you know what you are doing?
```

Reply:
I found that you can now get it to fail if you have an install/use mismatch, which I think is better. If no installation is selected, then no error will be thrown.